src/metrics.rs

```diff
@@ -226,6 +226,23 @@ pub fn register_process_metrics(_registry: &Registry) -> prometheus::Result<()>
     Ok(())
 }
 
+pub fn register_build_info_metrics(registry: &Registry) -> prometheus::Result<()> {
+    let opts =
+        Opts::new("build_info", "Build information about the server").namespace("outpack_server");
+    let labels = ["version", "revision", "rustc"];
+    let values = [
+        env!("CARGO_PKG_VERSION"),
+        env!("VERGEN_GIT_SHA"),
+        env!("VERGEN_RUSTC_SEMVER"),
```

Comment:
these are set by default im assuming?

Reply:
`CARGO_PKG_VERSION` is set automatically by Cargo. The `VERGEN_*` ones are set by the build script, using the vergen crate.

Note that `env!` looks up environment variables at build time, not runtime.